resources/kcp/charts/kyma-environment-broker/templates/service-monitor.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
 spec:
   endpoints:
    - port: http
+    interval: 20
+    scrapeTimeout: 20
```

Comment:
Could you please make these configurable similar to this:
https://github.com/kyma-project/control-plane/blob/main/resources/kcp/charts/mothership-reconciler/templates/service-monitor.yaml#L12-L13

The default value could be `30s` for the `interval` and `10s` for the `scrapeTimeout`